packages/interaction-kit/src/application.ts

```diff
@@ -131,71 +134,54 @@ export default class Application {
 		return this.#components.get(customID);
 	}
 
-	getCommand(type: ApplicationCommandType, customID: string) {
-		return this.#commands.get(type).get(customID);
+	getCommand(type: ApplicationCommandType, name: string) {
+		return this.#commands.get(type).get(name);
 	}
 
-	// TODO: Should this be moved into Command?
-	async updateCommands() {
-		console.log("Checking for command updates in Development Server");
-
-		if (!process.env.DEVELOPMENT_SERVER_ID) {
-			throw new NoDevelopmentServerEnvironmentVariableError();
-		}
+	loadApplicationCommandDirectory(directoryPath: string) {
+		const directory = path.join(
+			path.dirname(url.fileURLToPath(import.meta.url)),
+			directoryPath
+		);
 
-		const guildID: Snowflake = process.env.DEVELOPMENT_SERVER_ID as Snowflake;
-		const json = await API.getGuildApplicationCommands(guildID);
+		fs.readdir(directory, async (err, files) => {
+			if (err) {
+				throw err;
+			}
 
-		// TODO: Handle errors
-		/**
-		 * Not in development server:
-		 *  { message: 'Missing Access', code: 50001 }
-		 */
+			for (const file of files) {
+				if (file.endsWith(".js")) {
```

Comment:
I feel like this is a bug

Reply:
This shouldn't be a problem unless there are files in the command directory which don't contain commands (helper files for example).

Reply:
Yeah, the plan was to have a `lib` dir for that code - I should add that to the template with a .gitkeep probably